app/views/producers/_fat.html.haml

```diff
@@ -75,7 +75,7 @@
     .row.cta-container
       .columns.small-12
         %a.cta-hub{"ng-repeat" => "hub in producer.hubs | visible | orderBy:'-active'",
-        "ng-href" => "{{::hub.path}}", "ofn-change-hub" => "hub",
+        "ng-href" => "{{::hub.path}}", target: @target, "ofn-change-hub" => "hub",
```

Comment:
I would prefer a helper here like `embedable_link_target`. What do you think @oeoeaio?

Reply:
From what I understand is the controller the place where we can check if it's embedded or not, right?

Reply:
Hm, you are right about that. I'm okay with this solution, I just wish there would be a cleaner way of separating UI and logic.